cluster/cluster.yaml

```diff
@@ -556,7 +556,7 @@ Resources:
                 Resource: '*'
             Version: 2012-10-17
           PolicyName: root
-      RoleName: "{{.Cluster.LocalID}}-app-autoscaler"
+      RoleName: "cluster-autoscaler-{{ .Cluster.Alias }}"
```

Comment:
We should keep the `app-` prefix as a kind of namespace to avoid clashing with other resources.